orderbook/telemetry/telemetry.go

```diff
@@ -0,0 +1,56 @@
+package telemetry
```

Comment:
Would introducing Orderbook module level telemetry package help to lower conginitive load comparing of global one? @p0mvn